exporter/sapmexporter/examples/signalfx-k8s.yaml

```diff
@@ -85,6 +85,9 @@ data:
           receivers: [signalfx, prometheus]
           processors: [memory_limiter, batch]
           exporters: [signalfx]
+        logs:
+          receivers: [signalfx]
```

Comment:
We should probably also add `processors: [memory_limiter, batch]` for uniformness.